src/explorer.ts

```diff
@@ -196,14 +205,28 @@ export class OpenShiftExplorer implements TreeDataProvider<ExplorerItem>, Dispos
                     description: 'Repositories',
                     iconPath: path.resolve(__dirname, '../../images/context/helm.png')
                 }
+            } else if (element.kind === 'Pod') {
+                const contextElement: DeploymentPodObject = element;
+                return {
+                    contextValue: 'openshift.k8sobject.Deployment.pod',
+                    label: contextElement.metadata.name,
+                    description: `${contextElement.kind.substring(0, 1).toLocaleUpperCase()}${contextElement.kind.substring(1)}`,
+                    collapsibleState: TreeItemCollapsibleState.None,
+                    iconPath: new ThemeIcon('layers-active'),
+                    tooltip: `${contextElement.status.phase} (1/1)\n${contextElement.status.podIP}`,
```

Comment:
I don't think it's worth listing (1/1) if this value is hard coded. For example, what if pod contains a sidecar container, so there should be two containers listed?

Also, you filter out pods that are in the "Running" phase, so the first part will always be "Running"

Reply:
I also don't get the meaning of that hard coded `(1/1)` string in pod's tooltip.

Maybe this is a place that later is to be changed to something like `(pod #x of y)` or something like this, but if not, I don't see any point in hard coding it here.

Reply:
There were no possibilities of getting the number of `pod` on running state. So removed the `(1/1)` hardcoded value